File: ALGO-SET-1/8-Roy and Profile Picture/TestClass.java
import java.util.Scanner;
class TestClass
{
  public static void main(String[] args) {


  Scanner sc=new Scanner(System.in);
  int L=sc.nextInt();
  int N=sc.nextInt();
  for(int i=0;i<N;i++)
  {
    int W=sc.nextInt();
    int H=sc.nextInt();

     if(W<L||H<L)
     {
       System.out.println("UPLOAD ANOTHER");
     }
     else if(W>=L&&H>=L)
     {
       if(W==L&&H==L)
       {
             System.out.println("ACCEPTED");
       }
       else if(W==H)
       {
          System.out.println("ACCEPTED");
       }
       else
       {
         System.out.println("CROP IT");
       }
     }


  }

  }
}

File: ALGO-SET-1/8-Roy and Profile Picture/TestClass.py
L=int(input())
N=int(input())
for i in range(0,N):
    data=input()
    W,H=data.split()
    W=int(W)
    H=int(H)
    if W<L and H<L:
        print("UPLOAD ANOTHER")
    elif W>=L and H>=L:
        if W==L and H==L:
            print("ACCEPTED")
        elif W==H:
            print("ACCEPTED")
        else:
            print("CROP IT")
    else:
        print("UPLOAD ANOTHER")
